doc/source/_static/css/examples.css

```diff
@@ -0,0 +1,199 @@
+:root {
+  --ray-example-gallery-gap-x: 18px;
+  --ray-example-gallery-gap-y: 22px;
+}
+
+#site-navigation {
+  width: 330px !important;
```

Comment:
nit: why do we need !important here?

Reply:
There's a ton of places in this PR that override sphinx css rules or rules in `_static/css/custom.css`. I did what I could to avoid using `!important`, but this was the only way I got the rule to apply. If sphinx offers some other override mechanism on a per-page basis, I'd be happy to make use of it, but since we're injecting styles and modifying content on the fly with `_static/js/tags.js`, I'm guessing there's no simple way to do it.

Reply:
In this particular case, this rule overrides one in `_static/css/custom.css` :shrug:

Reply:
Nice! Approved